.github/workflows/linux.yml

```diff
@@ -91,8 +91,18 @@ jobs:
           set -x
           git fetch --prune --unshallow
```

Comment:
I really hoped we can drop this line now and unshallow as an option to checkout action… but there's no such option, and GitHub recommends using this line instead (not nice). But it means using it in this place (close to the actual line, that needs it) is OK.